packages/tpetra/core/ext/TpetraExt_MatrixMatrix_def.hpp

```diff
@@ -561,7 +562,6 @@ makeColMapAndConvertGids(GlobalOrdinal ncols,
   return rcp(new map_type(Teuchos::OrdinalTraits<GlobalOrdinal>::invalid(), colmap, 0, comm));
```

Comment:
Technically should be `Teuchos::OrdinalTraits<Tpetra::global_size_t>::invalid()` but that's OK :)